tests/validation/cattlevalidationtest/core/common_fixtures.py

```diff
@@ -2803,9 +2803,10 @@ def create_ingress(file_name, ingress_name, namespace,
 
 
 def wait_for_ingress_to_become_active(ingress_name, namespace):
-    lb_ip = None
+    # Returns a list of lb_ips [Supports ingress scaling]
+    lb_ip = []
     startTime = time.time()
-    while lb_ip is None:
+    while not lb_ip:
```

Comment:
We should wait for scale number of  lb_ips to be available here. This method should take scale as optional parameter defaulting to 1.